src/client/pages/Welcome.tsx

```diff
@@ -16,21 +17,25 @@ type Props = {
 };
 
 export const Welcome = ({ submitUrl, email, fieldErrors }: Props) => {
+  const autoRow = getAutoRow(1, gridItemColumnConsents);
   return (
-    <>
-      <Header />
-      <Main subTitle="Welcome">
-        <PageBox>
-          <PageHeader>Welcome</PageHeader>
-          <PageBody>
-            <PageBodyText>
-              Please enter your new password for {email}
-            </PageBodyText>
-            <PasswordForm submitUrl={submitUrl} fieldErrors={fieldErrors} />
-          </PageBody>
-        </PageBox>
-      </Main>
-      <Footer />
-    </>
+    <ConsentsLayout
+      title="Welcome to the Guardian"
+      current={CONSENTS_PAGES.PASSWORD}
+      bgColor={CONSENTS_MAIN_COLOR}
+      showContinueButton={false}
+    >
+      <ConsentsContent>
+        <p css={[text, autoRow()]}>
+          Please create a password for {email || 'your new account'}
```

Comment:
This reads really nice but isn't email required so should always be present? (Not a blocker!)

Reply:
Actually I'll rethink the logic around this! I think email should be optional, as there may be cases where a user accidently lands on the page directly and the email is no longer in the cookie!

Reply:
Good point!